web/src/todo_widget.ts

```diff
@@ -507,21 +671,62 @@ export function activate({
         }
 
         switch (last_event.type) {
-            case "new_task":
-                $elem
-                    .find("ul.todo-widget")
-                    .append($(render_widgets_todo_widget_tasks({all_tasks: [task]})));
+            case "new_task": {
+                const $html = $(render_widgets_todo_widget_tasks({all_tasks: [task]}));
+                $html.find(".todo-task-list-item-bar").toggle();
```

Comment:
Again, I think just using `toggle()` doesn't make it clear what's the exact behaviour that we want to achieve. So, this needs to be audited everywhere here.